heptapod.cabal

```
@@ -48,7 +48,7 @@ library
   build-depends:
     bytestring ^>=0.11.5.3 || ^>=0.12.1.0,
     entropy ^>=0.4.1.10,
-    time ^>=1.12.2 || ^>=1.14,
+    time ^>=1.12.2 || ^>=1.15,
```

I think we want to retain support for `time-1.14.*`:
```suggestion
    time ^>=1.12.2 || ^>=1.14 || ^>=1.15,
```